bin/MadGraph5_aMCatNLO/cards/production/13TeV/MuMuJet_mll_0to60_LO/dyellell01234j_5f_LO_MLM_run_card.dat

```diff
@@ -0,0 +1,270 @@
+# Use >=2 if most helicities contribute, e.g. pure QCD.
+#*************************************************************
+   0  = nhel          ! Number of helicities used per event
+#*******************                                                 
+# Standard Cuts
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Minimum and maximum pt's (for max, -1 means no cut)                *
+#*********************************************************************
+ 20  = ptj       ! minimum pt for the jets 
+  0  = ptb       ! minimum pt for the b 
+ 20  = pta       ! minimum pt for the photons 
+3.0  = ptl       ! minimum pt for the charged leptons 
+  0  = misset    ! minimum missing Et (sum of neutrino's momenta)
+  0  = ptheavy   ! minimum pt for one heavy final state
+  0  = ptonium   ! minimum pt for the quarkonium states
+ -1  = ptjmax    ! maximum pt for the jets
+ -1  = ptbmax    ! maximum pt for the b
+ -1  = ptamax    ! maximum pt for the photons
+ -1  = ptlmax    ! maximum pt for the charged leptons
+ -1  = missetmax ! maximum missing Et (sum of neutrino's momenta)
+#*********************************************************************
+# Minimum and maximum E's (in the center of mass frame)              *
+#*********************************************************************
+  0  = ej     ! minimum E for the jets 
+  0  = eb     ! minimum E for the b 
+  0  = ea     ! minimum E for the photons 
+  0  = el     ! minimum E for the charged leptons 
+ -1   = ejmax ! maximum E for the jets
+ -1   = ebmax ! maximum E for the b
+ -1   = eamax ! maximum E for the photons
+ -1   = elmax ! maximum E for the charged leptons
+#*********************************************************************
+# Maximum and minimum absolute rapidity (for max, -1 means no cut)   *
+#*********************************************************************
+   3  = etaj    ! max rap for the jets 
+  -1  = etab    ! max rap for the b
+   3  = etaa    ! max rap for the photons 
+   3  = etal    ! max rap for the charged leptons 
+  -1  = etaonium ! max rap for the quarkonium states
+   0  = etajmin ! min rap for the jets
+   0  = etabmin ! min rap for the b
+   0  = etaamin ! min rap for the photons
+   0  = etalmin ! main rap for the charged leptons
+#*********************************************************************
+# Minimum and maximum DeltaR distance                                *
+#*********************************************************************
+ 0   = drjj    ! min distance between jets 
+ 0   = drbb    ! min distance between b's 
+ 0   = drll    ! min distance between leptons 
+ 0   = draa    ! min distance between gammas 
+ 0   = drbj    ! min distance between b and jet 
+ 0.1 = draj    ! min distance between gamma and jet 
+ 0.4 = drjl    ! min distance between jet and lepton 
```

Comment:
I would suggest removing this cut to be safe from smearing

Reply:
Could you point out which cut specifically?

Reply:
I meant this drjl

Reply:
We put this cut on purpose. Our analysis selection is actually not DY+jet, but it is DY+gamma, where gamma comes from jet. So we want to have a sample with a jet, which would fake as a photon. Further, we want this jet to separate from the muons in angle. We checked that if this drjl cut was not imposed, a significant amount of events would have overlapped hard leptons and jet, which we do not want to have.

Reply:
That makes sense